dask/dataframe/io/parquet.py

```diff
@@ -0,0 +1,164 @@
+import struct
+
+from toolz import first
+
+from dask import delayed, compute
+from dask.bytes.core import OpenFileCreator
+import dask.dataframe as dd
+
+try:
+    import fastparquet
+    from fastparquet import parquet_thrift
+    default_encoding = parquet_thrift.Encoding.PLAIN
+except:
+    fastparquet = False
+    default_encoding = None
+
+
+def read_parquet(path, columns=None, filters=[], categories=None, index=None,
+                 **kwargs):
+    """
+    Read Dask DataFrame from ParquetFile
+
+    This reads a directory of Parquet data into a Dask.dataframe, one file per
+    partition.  It selects the index among the sorted columns if any exist.
+
+    Parameters
+    ----------
+    path : string
+        Source directory for data.
+        Prepend with protocol like ``s3://`` or ``hdfs://`` for remote data.
+    columns: list or None
+        List of column names to load
+    filters: list
+        List of filters to apply, like ``[('x', '>' 0), ...]``
+    index: string or None
+        Name of index column to use if that column is sorted
+
```

Comment:
The `categories` parameter isn't included here. Also `filters` defaults to an empty list, which can cause issues with mutability shared across function alls.